core/native/src/files/FileSystemNative.kt

```diff
@@ -125,6 +127,7 @@ internal actual val isWindows: Boolean = Platform.osFamily == OsFamily.WINDOWS
 @OptIn(ExperimentalStdlibApi::class)
```

Comment:
`ExperimentalStdlibApi` is not needed anymore in a lot of places where `AutoCloseable` is used (as well as `AutoCloseableAlias` is not needed anymore)
It would be nice to cleanup this before release (may be via separate PR)

Reply:
Thanks for reminding me about that! I think it fine to clean it up here